modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -1,77 +1,118 @@
 See link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
 +
-[cols="2,4,2",options="header"]
+[cols="3,4,1",options="header"]
 |===
-|URL | Function | Requirement
+|URL | Function | Port
 
 |`https://cloud.redhat.com`
-|Used by the cluster for the `insights operator` that integrates with the SaaS Red Hat Insights
-|Required
+|Required. Used by the cluster for the `insights operator` that integrates with the SaaS Red Hat Insights.
+|443, 80
+
+|`cert-api.access.redhat.com`
+|Required. Used by telemetry.
+|443, 80
+
+|`api.access.redhat.com`
+|Required. Used by telemetry.
+|443, 80
+
+|`infogw.api.openshift.com`
+|Required. Used by telemetry.
+|443, 80
 |===
 
 . For Amazon Web Services (AWS), you must grant access to the URLs that provide the AWS API and DNS services:
 * You can grant access by allowing the `.amazonaws.com` wildcard:
 +
-[cols="2,4,2",options="header"]
+[cols="3,4,1",options="header"]
 |===
 |URL | Function | Requirement
 
 |`.amazonaws.com`
-|Required to access AWS services and resources.
-|Required
+|Required. Used to access AWS services and resources.
+|443, 80
+
+|`oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com`
+|Required. Used to access AWS services and resources when using strict security requirements. Review the AWS Service Endpoints in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
+|443, 80
 |===
 +
 * Alternatively, you can grant access by allowing the following regional AWS service endpoints:
 +
-[cols="2,4,2",options="header"]
+[cols="3,4,1",options="header"]
 |===
-|URL | Function | Requirement
+|URL | Function | Port
 |`ec2.<aws_region>.amazonaws.com`
-|Required for regional access to Amazon EC2 services. EC2 instances are required to deploy the control plane and data plane functions of ROSA. Replace `<aws_region>` with an AWS region code, for example `us-east-1`.
-|Required
+|Required. Used for regional access to Amazon EC2 services. EC2 instances are required to deploy the control plane and data plane functions of ROSA. Replace `<aws_region>` with an AWS region code, for example `us-east-1`.
+|443, 80
 
 |`elasticloadbalancing.<aws_region>.amazonaws.com`
-|Required to access Amazon Elastic Load Balancers (ELBs) for API and application load balancing.
-|Required
+|Required. Used to access Amazon Elastic Load Balancers (ELBs) for API and application load balancing.
+|443, 80
 
 |`<cluster_id>-<shard>.<aws_region>.amazonaws.com`
-|Required for access to the registry. Replace `<cluster_id>`, `<shard>`, and `<aws_region>` with your cluster ID, shard name and AWS region code.
-
-|Required
+|Required. Used for access to the registry. Replace `<cluster_id>`, `<shard>`, and `<aws_region>` with your cluster ID, shard name and AWS region code.
```

Comment:
we should explicity call out that these details wont be available until post/during installation? and how to actually get them?

Reply:
@dofinn  How do they get them?

Reply:
Please see if you agree with what I've drafted. Thanks.

Reply:
/lgtm